shopinvader_backend_image_proxy/models/shopinvader_backend.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ShopinvaderBackend(models.Model):
+    _inherit = 'shopinvader.backend'
+
+    image_proxy_url = fields.Char(
+        help="Replace the original url (base part) by this proxy url during "
+             "export of images serialized field.\n"
+             "Fill without specify the protocol and anything else than the "
+             "complete website name (with subdomain if any).\n"
```

Comment:
s/Fill without specify the protocol and anything else/Fill without specifying the protocol or anything else